TipCatDotNet.Api/Data/Models/User.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace TipCatDotNet.Api.Data.Models
+{
+    [Table("users")]
+    public class User
```

Comment:
Why can't we merge both owners and employees to the one `Employee` table? There aren't much differences